servant/server/src/Mu/Servant/Server.hs

```diff
@@ -0,0 +1,351 @@
+{-# language DataKinds             #-}
+{-# language DeriveGeneric         #-}
+{-# language FlexibleContexts      #-}
+{-# language FlexibleInstances     #-}
+{-# language GADTs                 #-}
+{-# language MultiParamTypeClasses #-}
+{-# language PolyKinds             #-}
+{-# language RankNTypes            #-}
+{-# language ScopedTypeVariables   #-}
+{-# language TypeApplications      #-}
+{-# language TypeFamilies          #-}
+{-# language TypeOperators         #-}
+{-# language UndecidableInstances  #-}
+{-# OPTIONS_GHC -Wall #-}
```

Comment:
We prefer to have this line as part of the `.cabal` file instead of per-file.

Reply:
Done!